mcux/mcux-sdk-ng/drivers/common/fsl_common_arm.h

```diff
@@ -474,15 +474,21 @@ _Pragma("diag_suppress=Pm120")
 #endif
 
 #elif (defined(__GNUC__)) || defined(DOXYGEN_OUTPUT)
+#if defined(__ARM_ARCH_8A__) /* This macro is ARMv8-A specific */
+#define __CS "//"
+#else
+#define __CS "@"
```

Comment:
Hello, because of MISRA C 2012 rule 21.1: https://stackoverflow.com/questions/50696794/misra-c-2012-rule-21-1-macros-starting-with-underscore. We will not use macro name started with underscore.
Also, this file will be included by every SDK file, to make it not conflict with other code, suggest adding prefix, like `MCUX_CS`. BTW, what is the meaning of CS? thanks.

Reply:
CS is comment sign character, so let's change it to be "MCUX_CS", thanks.